src/components/Button/Button.stories.tsx

```diff
@@ -117,52 +120,45 @@ SecondarySmall.args = { variant: 'secondary', size: 'sm' };
 
 export const Tertiary = Template.bind({});
 Tertiary.args = {
-  children: 'Button',
-  variant: 'secondary',
   status: 'neutral',
 };
 
-export const SecondaryNeutralDisabled = Template.bind({});
```

Comment:
I had renamed "SecondaryNeutral" to "Tertiary" but forgot to update the others, so I cleaned that up in here as well.